packages/terra-embedded-content-consumer/src/clinical-lowlight-theme/EmbeddedContentConsumer.module.scss

```diff
@@ -0,0 +1,6 @@
+:local {
+  .clinical-lowlight-theme {
+    --terra-embedded-content-consumer-focus-outline: 2px dashed #b2b5b6;
+    --terra-embedded-content-consumer-blur-outline: none;
```

Comment:
We shouldn't need a blur outline style.  That should just be the normal state of the element.

Reply:
Are you saying we can hard-code `outline: none` to remove the style from the frame when there is no focus anymore? Because we set the `outline` style when there is focus. And if we don't remove that style, it stays.

Reply:
No, I am saying that you would have CSS for the normal state.  When the element is not focused, that CSS will no longer be applied.